docs/internal/release-procedure.md

```diff
@@ -174,3 +174,127 @@ git push origin version-<YOUR_VERSION>
 ```
 
 replacing `<YOUR_VERSION>` with the version you want. Then create a PR and merge it.
+
+## Manual build
+
+Creating the release for Ubuntu 18.04 ([arduino/arduino-ide#2018](https://github.com/arduino/arduino-ide/issues/2018)) and macOS M1 ([arduino/arduino-ide#408](https://github.com/arduino/arduino-ide/issues/408)) is a manual procedure.
+
+### Ubuntu 18.04
+ - Prerequisites:
+    - Ask the DevOps team for an EC2 instance with at least 8 GB of RAM.
+    - Your account must have access to the staging environment.
+    - You have VPN connection to staging.
+ - Setup:
+    - To install all required dependencies, run the following script:
+       ```sh
+       sudo apt update \
+       && sudo apt install --no-install-recommends --yes \
+         git \
+         gcc \
+         curl \
+         make \
+         python \
+         pkg-config \
+         libx11-dev \
+         libxkbfile-dev \
+         build-essential \
+         libsecret-1-dev \
+       && wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash \
+       && source ~/.bashrc \
+       && nvm install 16 \
+       && nvm use 16 \
+       && nvm alias default 16 \
+       && curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add - \
+       && echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list \
+       && sudo apt update && sudo apt install --no-install-recommends yarn
+       ```
```

Comment:
Add instructions for cloning the `arduino/arduino-ide` repository into the EC2 instance machine and checking out the release tag.